docs/GettingStarted.md

```diff
@@ -2257,3 +2258,60 @@ However, additional instrumentation provided by Datadog can be activated alongsi
 | `OpenTracing::FORMAT_TEXT_MAP` | Yes        |                        |
 | `OpenTracing::FORMAT_RACK`     | Yes        | Because of the loss of resolution in the Rack format, please note that baggage items with names containing either upper case characters or `-` will be converted to lower case and `_` in a round-trip respectively. We recommend avoiding these characters or accommodating accordingly on the receiving end. |
 | `OpenTracing::FORMAT_BINARY`   | No         |                        |
+
+### Profiling
+
+*Currently available as BETA feature.*
+
+`ddtrace` can produce profiles that measure method-level application resource usage within production environments. These profiles can give insight into resources spent in Ruby code outside of existing trace instrumentation.
+
+**Compatibility**
+
+Basic profiling requires:
+
+- MRI Ruby 2.0+
+- `google-protobuf` gem installed
+
+In addition, CPU time measurements require:
+
+- MRI Ruby 2.1+
+- Linux OS with `pthread` support
+- `ffi` gem installed
+
+**Setup**
+
+1. Add gems to your `Gemfile` and `bundle install`:
+
+    ```ruby
+    gem 'ddtrace'
+    gem 'google-protobuf'
+    gem 'ffi'
+    ```
+
+2. Configure profiling (for Rails applications `config/initializers/datadog.rb`):
+
+    ```ruby
+    Datadog.configure do |c|
+      c.profiling.enabled = true
+    end
+
+    # Start the profiler
+    Datadog.profiler.start
```

Comment:
Should we alert the user if the profile was not started with the `ddtracerb` preloader?

Reply:
No, this is acceptable as you can still receive wall timing for profiling. Might be good for some environments that don't support pthread.